Week1/exercise1.js

```diff
@@ -0,0 +1,22 @@
+var mysql = require('mysql');
```

Comment:
The queries are correct, but you are not supplying a callback so it won't know if a query fails and returns an error.